job_templates/packaging_job.xml.em

```diff
@@ -238,10 +238,15 @@ rmdir /S /Q ws workspace
 
 set CONTAINER_NAME=ros2_windows_ci_%CI_ROS_DISTRO%
 set DOCKERFILE=windows_docker_resources\Dockerfile
+set SOLOFILE=windows_docker_resources\install_ros2_%CI_ROS_DISTRO%.json
+set VISUAL_STUDIO_VERSION=%CI_VISUAL_STUDIO_VERSION%
 
 rem "Change dockerfile once per day to invalidate docker caches"
 powershell "(Get-Content ${Env:DOCKERFILE}).replace('@@todays_date', $(Get-Date).ToLongDateString()) | Set-Content ${Env:DOCKERFILE}"
 
+rem "Change the solo install JSON file to contain the Visual Studio version we want to use
```

Comment:
```suggestion
rem "Change the chef-solo configuration file to set a specific the Visual Studio version.
```

I got hung up on 'solo install JSON file' so I attempted a re-word.

Reply:
Fixed in [b7ae4a5](https://github.com/ros2/ci/pull/786/commits/b7ae4a59d079670a8f7a8772d5ba0b3102013732)